lib/testsuite/index.js

```diff
@@ -762,7 +762,7 @@ class TestSuite {
           return this.retryCurrentTestCase();
         }
 
-        if ((possibleError instanceof Error) && this.shouldSkipTestsOnFail()) {
+        if ((possibleError instanceof Error) && possibleError.abortOnFailure && this.shouldSkipTestsOnFail()) {
```

Comment:
I think we should rather keep the `verify` error limited to `asynctree.js`. Over here the error could be from creating a test session and as this is not an instance of NightwatchAssertError it would not have `abortOnfailure` key